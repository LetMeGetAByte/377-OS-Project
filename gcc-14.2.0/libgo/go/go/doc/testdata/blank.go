version https://git-lfs.github.com/spec/v1
oid sha256:a8d0b60bcb0ec29c689fb19806713e94044014f9c998168f8d8963ac7529e25c
size 1076
