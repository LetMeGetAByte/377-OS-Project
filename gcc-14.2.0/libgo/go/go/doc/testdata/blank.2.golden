version https://git-lfs.github.com/spec/v1
oid sha256:0214a19df93fe078d8ce86b57f9fa0fd672f796ebc4723f426c11a7ed6b8ebda
size 751
